.github/workflows/lint.yaml

```
@@ -10,19 +10,19 @@ jobs:
 
     steps:
       - name: Check out git repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Set up Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v4
         with:
-          node-version: 12
+          node-version: 20
 
       # ESLint must be in `package.json`
       - name: Install Node.js linting dependencies
         run: npm ci
 
       - name: Run linters
-        uses: wearerequired/lint-action@v1
+        uses: rkuykendall/lint-action@master
```

I think you switched to this fork for node 20 support. looks like a good fix, but the new author is a bit of an unknown since they're an individual vs a company (wearerequired). unclear if the author will maintain this fork going forward, or what direction they'll take it in.

they didn't release any git tags, so I used a git commit hash to make sure any new versions don't break CI unexpectedly. there's also [some security reasons to use a git hash](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions), but I'm not convinced we need to make the same change everywhere yet.

let me know if I'm missing something about though

```suggestion
        # TODO switch back to wearerequired/lint-action when node 20 supported
        uses: rkuykendall/lint-action@3402def6dea8dbcf0b8ec42431d9345ee3f69c4d
```